Oqtane.Client/Services/FolderService.cs

```diff
@@ -37,11 +37,11 @@ public async Task<Folder> GetFolderAsync(int siteId, [NotNull] string folderPath
         {
             if (!(folderPath.EndsWith(System.IO.Path.DirectorySeparatorChar) || folderPath.EndsWith(System.IO.Path.AltDirectorySeparatorChar)))
             {
-                folderPath = Utilities.PathCombine(folderPath, "\\");
+                folderPath = Utilities.PathCombine(folderPath, System.IO.Path.DirectorySeparatorChar.ToString());
```

Comment:
Could you please provide two unit tests for `FileService` & `FolderService` to ensure nothing is broken?